ccd-definition/CaseEventToComplexTypes/returnApplication/ReturnApplication.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ReturnApplication",
+    "CaseEventID": "returnApplication",
+    "CaseFieldID": "returnApplication",
+    "ListElementCode": "reason",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ReturnApplication",
```

Comment:
are these names a bit generic, do we want to go with returnNote or what is your thoughts? I'm thinking outside the context of this json file it might be a bit unclear..